### PR TITLE
add filter to only call picturefill on elements in viewport

### PR DESCRIPTION
**Pull request by brycedorn**
# Desc

untested; unsure how to test betty

initial approach was to have an env var for `DOCUMENT_FOLD_HEIGHT` but this felt inconsistent (especially across different browsers/window sizes); would rather have this be determined dynamically.

### CODE REVIEW

File: betty/cropper/templates/image.js [hunk 1]
```diff
@@ -33,6 +33,19 @@
     };
   }
 
+  // Credit to http://ejohn.org/blog/getboundingclientrect-is-awesome/
+  function isInViewport(el) {
```
Comment (spra85) [inline betty/cropper/templates/image.js]: Worth re-using this somehow? https://github.com/theonion/bulbs-elements/blob/master/lib/bulbs-elements/util/in-view-monitor.js

/cc @collin

Comment (collin) [inline betty/cropper/templates/image.js]: Hm, there's not a simple way to get that function in to this js file. Probably a low ROI for re-use.

**Comment by collin** [comment]
@brycedorn there's already a spot that is checking the element viewport location, maybe we could make the change here, or at least remove one of the checks:

https://github.com/theonion/betty-cropper/blob/09e7f0f5a30f76ff1c5953f9d486b144cfcb9e44/betty/cropper/templates/image.js#L75-L80

I think the deficiency with that check is that any image inside or above the viewport will load. Images below are skipped.

**Comment by brycedorn** [comment]
@collin you're correct, was going to see if this had any effect before moving forward.

didn't look deep enough into that loop, I see now. this task seems redundant to me now - may need further discussion if moving the call to `fill` will have a high enough impact (vs changing how other elements are loaded on the page)

**Comment by mparent61** [comment]
@brycedorn I'll show you how to setup local + test environments when I'm back in the office, probably Thursday

**Comment by brycedorn** [comment]
@mparent61 thanks! may not be necessary, though. need clarification re: above ☝️

**Comment by brycedorn** [comment]
closing this per discussion; this approach already implemented, change only needed in omni

**Pull request closed without merging**